includes/functions.php

```diff
@@ -580,6 +580,7 @@ function createHost($host, $community = NULL, $snmpver, $port = 161, $transport
         'port' => $port,
         'transport' => $transport,
         'status' => '1',
+        'ip' => $ip,
```

Comment:
Sorry, I don't get it. In the device table, 'ip' field represent indeed the ip of the device. What do you mean by 'converting', I should convert this ip into what ? 
$ip is filled and checked before being recorded in database, regarding the following code : 
```
...
 256         if ($config['addhost_alwayscheckip'] === TRUE) {
 257             $ip = gethostbyname($host);
 258         } else {
 259             $ip = $host;
 260         }
 261         if (ip_exists($ip) === false) {
...
```